subsys/shell/Kconfig

```diff
@@ -25,6 +25,20 @@ config SHELL_MINIMAL
 	  defaults which favor reduced flash or memory requirements over extra
 	  features.
 
+config SHELL_THREAD_PRIORITY_OVERRIDE
+	bool "Override default shell thread priority"
+	default n
```

Comment:
By convention, we do not specify `default n` because it is implied.

Reply:
Thank you. Fixed.